roles/ovirt-vm-infra/README.md

```diff
@@ -40,6 +40,12 @@ The `profile` dictionary can contain following attributes:
 | domain             | UNDEF                 | The domain of the virtual machine.                         |
 | root_password      | UNDEF                 | The root password of the virtual machine.                      |
 | high_availability  | UNDEF                 | Whether or not the node should be set highly available. |
+| storage_domain     | UNDEF                 | Name of the storage domain where all template disks should be created. |
+|                    |                       | This parameter is considered only when template is provided. |
+|                    |                       | This parameter is not idempotent, if the VM exists and you specfiy different storage domain,|
+|                    |                       | disk won't move. |
+|                    |                       | ansible bersion version: "2.4"|
```

Comment:
So this will crash with 2.3, that's not good, I would just add the latest `ovirt_vms`, into the `library` directory. If it's compatible with latest `ovirt.py` from module_utils.